src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -348,7 +408,38 @@ END_SECTION
 
 START_SECTION(getParameters().getDescription("rt_window"))
 {
-  TEST_EQUAL(ptr->getParameters().getDescription("rt_window"), "Retention time window.")
+  TEST_EQUAL(ptr->getParameters().getDescription("rt_window"), "Retention time window in seconds.")
+}
+END_SECTION
+
+START_SECTION(pickSpectrum())
+{
+  // TODO replace and improve this test
```

Comment:
You could add in a test for the m/z and intensity for the identified peaks.

Reply:
created an issue for this: https://github.com/biosustain/OpenMS/issues/14